Gems/Multiplayer/Code/Include/Multiplayer/NetworkEntity/INetworkEntityManager.h

```diff
@@ -89,6 +89,11 @@ namespace Multiplayer
         //! @return the total number of entities tracked by this INetworkEntityManager instance
         virtual uint32_t GetEntityCount() const = 0;
 
+        //! Returns the Net Entity ID for a given AZ Entity ID.
+        //! @param entityId the AZ Entity ID
+        //! @return the Net Entity ID
+        virtual NetEntityId GetNetEntityIdById(const AZ::EntityId& entityId) const = 0;
```

Comment:
I was about to say you don't need to const-ref the AZ::EntityId since it's just an integral type, but then I checked and it's not :(  I'll have to start talking to core about making that a type-safe integral instead.